src/coreclr/src/jit/hwintrinsic.cpp

```diff
@@ -548,22 +599,24 @@ bool Compiler::compSupportsHWIntrinsic(InstructionSet isa)
 // impIsTableDrivenHWIntrinsic:
 //
 // Arguments:
+//    intrinsicId - HW intrinsic id
 //    category - category of a HW intrinsic
 //
 // Return Value:
 //    returns true if this category can be table-driven in the importer
 //
 static bool impIsTableDrivenHWIntrinsic(NamedIntrinsic intrinsicId, HWIntrinsicCategory category)
 {
-    return (category != HW_Category_Special) && (category != HW_Category_Scalar) &&
-           HWIntrinsicInfo::RequiresCodegen(intrinsicId) && !HWIntrinsicInfo::HasSpecialImport(intrinsicId);
+    return (category != HW_Category_Special) && HWIntrinsicInfo::RequiresCodegen(intrinsicId) &&
+           !HWIntrinsicInfo::HasSpecialImport(intrinsicId);
 }
 
 //------------------------------------------------------------------------
 // impHWIntrinsic: Import a hardware intrinsic as a GT_HWINTRINSIC node if possible
 //
 // Arguments:
 //    intrinsic -- id of the intrinsic function.
+//    clsHnd     -- class handle containing the intrinsic function.
```

Comment:
Since you're here, you could also add the `mustExpand` argument to the comment.

Reply:
done.